packages/webpack/lib/configs/build.js

```diff
@@ -94,6 +93,10 @@ module.exports = function getConfig(config, name) {
       devtoolModuleFilenameTemplate: (info) =>
         relative(config.rootDir, info.absoluteResourcePath),
     },
+    // fixme
```

Comment:
What about the cache, why do we use the memory cache and not the disk cache? IIRC we also need to make sure that the cache is invalidated correctly when the config changes. I think it was by providing a list of file paths that affect the config, no?

Reply:
It's low prio imho. Might happen in a subsequent PR even.